network-api/networkapi/wagtailpages/templates/wagtailpages/tags/cta.html

```diff
@@ -3,7 +3,3 @@
 {% if cta_type == 'Petition' %}
 	{% include './cta/petition.html' %}
 {% endif %}
-
-{% if cta_type == 'Signup' %}
```

Comment:
I assume this also means removing the Signup CTA subclass, but in a different (follow-up) PR?

Reply:
This is for removing the `signup`. In that case, I'll remove basket in a separate pr and will also file a ticket to remove the signup cta subclass from `models.py`. 👍